delta-lake/delta-spark350db143/src/main/scala/com/nvidia/spark/rapids/delta/shims/UpdateCommandMetaShim.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.delta.shims
+
+import com.databricks.sql.transaction.tahoe.commands.DeletionVectorUtils
+import com.databricks.sql.transaction.tahoe.sources.DeltaSQLConf
+import com.nvidia.spark.rapids.delta.{UpdateCommandEdgeMeta, UpdateCommandMeta}
+
+object UpdateCommandMetaShim {
+  def tagForGpu(meta: UpdateCommandMeta): Unit = {
+    val deltaLog = Option(meta.updateCmd.tahoeFileIndex).map(_.deltaLog).orNull
+    val dvFeatureEnabled = deltaLog == null ||
```

Comment:
Why deltaLog == null would imply dvFeatureEnabled is true?

Reply:
It doesn't mean that it's enabled. If deltaLog is null, I want it to be insignificant in determining the condition that follows. 

I will update to make this better

Reply:
Updated. I see downstream code directly calling `deltaLog` on tahoeFileIndex object without guarding against null so we don't need to.